master/usage/routereflector/bird-rr-config.md

```diff
@@ -42,20 +42,43 @@ single `bird` package installs both IPv4 and IPv6 BIRD):
     sudo apt-get update
     sudo apt-get install bird
 
-#### RHEL 7
+#### RHEL
 
-First, install EPEL. Depending on your system, the following command may
-be sufficient:
+1. From a terminal prompt, create a new file called bird.repo in the 
+   `/etc/yum.repos.d/` directory.
 
-    sudo yum install epel-release
+   ```bash 
+   vi /etc/yum.repos.d/bird.repo
+   ```
+   
+   > **Note**: Creating a file in this directory requires root privileges.
+   {: .alert .alert-info}
 
-If that fails, try the following instead:
+1. Add the following lines to the file.
 
-    sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+   ```
+   [bird]
+   name=Network.CZ Repository
+   baseurl=ftp://repo.network.cz/pub/redhat/
+   enabled=1
+   gpgcheck=0
+   gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-network.cz
```

Comment:
We are missing a step here, to download the BIRD team's key and save it at `/etc/pki/rpm-gpg/RPM-GPG-KEY-network.cz`.  Please see http://bird.network.cz/?download&tdir=redhat/ for where to get that key from.